cms/templates/base.html

```diff
@@ -85,7 +86,7 @@
                       url_name: "${context_course.location.name | h}",
                       org: "${context_course.location.org | h}",
                       num: "${context_course.location.course | h}",
-                      display_course_number: "${_(context_course.display_coursenumber)}",
+                      display_course_number: "${_(HTML_ESCAPE(context_course.display_coursenumber))}",
```

Comment:
We can't use `| h` because  `display_coursenumber` is passed to `ugettext` as a parameter. Alright, I will right up a test as well.